README.md

```diff
@@ -86,6 +86,7 @@ Fields available in the `TalosConfigTemplate` (and `TalosConfig`) resources:
    It is recommended to always set this field explicitly to avoid issues when CABPT is upgraded to the version which supports new Talos version.
 - `configPatches` (optional): set of machine configuration patches to apply to the generated configuration.
 - `data` (only for `generateType: none`): user-supplied machine configuration.
+- `hostnameFromMachineName`: set the hostname in the generated machine configuration to the `Machine` name (not supported with `MachinePool` deployments)
```

Comment:
What about something like "GenerateHostname" and keeping it generic?

Reply:
I thought about something like

```yaml
spec:
   hostname:
      source: machineName
```

?

Reply:
I like this idea. We can expand it in different ways.